docs/application/native/guides/connectivity/stc.md

```diff
@@ -49,9 +49,9 @@ To enable your application to use the STC API:
    stc_deinitialize();
    ```
 
```

Comment:
I think we need `<a name="retrieve_data_usage_system"></a>` here. Please check and add it if it is needed.

Reply:
resolved, now link is working